loops.js

```diff
@@ -25,7 +35,15 @@
 // klaar!
 // ==========================================
 
+const word = ["loop...", "loop...", "loop...", "loop..."];
+// console.log(` ${word.repeat(4)}`);
 
+for (let i = 0; i < word.length; i++) {
+    console.log(word[i])
+}
+    if (word.length > 3) {
```

Comment:
Keurig de if statement gebruikt, maar waar is de else? 

Wanneer word.length niet groter is dan 3, dan wil je console.log('loop...') laten zien. Die had je dus perfect in de else{} kunnen zetten.